travis-ymls/react-router.travis.yml

```diff
@@ -0,0 +1,89 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : react-router
+# Source Repo         : https://github.com/ReactTraining/react-router
+# Travis Job Link     : https://travis-ci.com/github/dthadi3/react-router/builds/210998601
+# Created travis.yml  : No
+# Maintainer          : Devendranath Thadi <devendranath.thadi3@gmail.com>
+#
+# Script License      : MIT License, Version 2 or later
```

Comment:
Script license should always be Apache 2.0

Reply:
Done.